# rMCA : Control mode for Channel A
uint16 r_mca

# rVCA : Target vacuum for Channel A
uint16 r_vca

# rMCB : Control mode for Channel B
uint16 r_mcb

# rVCB : Target vacuum for Channel B
uint16 r_vcb